src/components/Article.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import CommentsList from './CommentsList'
+import Vote from './Vote';
+
+const Article = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
```

Comment:
Don't forget to set isLoading back to true at the beginning of useEffect in case an error occurs and requires re-rendering